.gush.yml

```diff
@@ -1,6 +1,7 @@
 adapter: github
 issue_tracker: github
 meta-header: "This file is part of Gush package.\n\n(c) 2013-2015 Luis Cordova <cordoval@gmail.com>\n\nThis source file is subject to the MIT license that is bundled\nwith this source code in the file LICENSE."
+pr_type: ['feature', 'bugfix', 'refactor', 'tests', 'remove', 'style', 'documentation', 'security']
```

Comment:
I think this list is a bit expensive :) basically you only need:

* feature (new feature)
* bug (programming bug fixed)
* style (style changes)
* doc (documentation)
* security (security bug/vulnerability fixed)

A missing test is a bug, remove? refactoring can be considered a feature but we can keep it :)
Remember that the longer the type gets the less you will see of the actual title, as first-line commits are always truncated to 50 chars.